### PR TITLE
[pytorch] Test key ET models export to core aten ir

### PR DESCRIPTION
**Pull request by guangy10**
Differential Revision: D48992081

### CODE REVIEW

File: test/test_model_exports_to_core_aten.py [hunk 1]
```diff
@@ -3,22 +3,28 @@
...
+        """
+        Check if all ops in the graph module are core aten ops.
+        """
+        verifier = ATenDialectVerifier()
+        for node in gm.graph.nodes:
+            if (
+                node.op == "call_function"
+                and node.target not in verifier.valid_builtin_funcs()
+            ):
+                with self.assertRaises(SpecViolationError):
```
Comment (larryliu0820) [inline test/test_model_exports_to_core_aten.py:23]: hmm I'm confused, should this be assertNotRaise?

Comment (kimishpatel) [reply test/test_model_exports_to_core_aten.py:23]: yeah same question. Also I dont think this verifier will sufficiently check for core aten ops, right?

**Review by kimishpatel — CHANGES_REQUESTED** [review]
Have you tried running this by disabling @larryliu0820's decomp? I would expect the test to fail.

**Comment by larryliu0820** [comment]
> Have you tried running this by disabling @larryliu0820's decomp? I would expect the test to fail.

Yeah I think we should create test that expect failure. This can be done by customizing decomp rules.

File: test/test_model_exports_to_core_aten.py (hunk 1 above)
Comment (guangy10) [reply test/test_model_exports_to_core_aten.py:23]: > hmm I'm confused, should this be assertNotRaise?

Yeah, it should be `assertNotRaises`.

> I dont think this verifier will sufficiently check for core aten ops, right?

Why? I'm using `ATenDialectVerifier`, by looking at it's impl, it is checking against core aten ops. https://github.com/pytorch/pytorch/blob/275e71c562f3f1e0232e2f4a22b5388e0e468864/torch/_export/verifier.py#L117-L135

> Have you tried running this by disabling @larryliu0820's decomp? I would expect the test to fail.

I would expect it to fail as well. Don't get a chance to work on this today, let me test it tomorrow.

Comment (kimishpatel) [reply test/test_model_exports_to_core_aten.py:23]: oh sorry I missed that. yes `ATenDialectVerifier` should do the job.

**Comment by guangy10** [comment]
Going to get back to this in IM3. We do want to protect the enabled models from breaking changes

**Pull request closed without merging**